controller/applicationsController.js

```diff
@@ -0,0 +1,67 @@
+const Admin = require("../models/admins");
+const Organization = require("../models/organizations");
+const Application = require("../models/applications");
+const { getAppToken } = require("../utils/auth/tokenGenerator");
+const { ObjectId } = require("mongoose").Types;
+
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+
+const createApplication = async (req, res, next) => {
+  try {
+    //validation should be done via middleware
+    //ownerId in body also needs to be validated
+
+    const { name, organizationId, createdBy } = req.body;
```

Comment:
The only property in `req.body` should be name for the application name. `orgnaizationId` and `createdBy` are gotten from the `req.token` that is decoded by the `orgAuthMW` as `organizationId` and `adminId` respectively. As stated prior the `createdBy` field is actually the `adminId`, which is found on req.token, of the authenticated admin who created the application